packages/forms/stories/8-Radio.stories.tsx

```diff
@@ -30,24 +31,27 @@ export const Default: Story<ICheckboxStoryProps & InputHTMLAttributes<HTMLInputE
     <Grid>
       <Row style={{ minHeight: 450 }}>
         <Col lg={4} offsetLg={4} md={6} offsetMd={3}>
-          <Field>
-            <Radio disabled={disabled} name="example">
-              <Label hidden={isHidden} isRegular={isRegular}>
-                Radio
-              </Label>
-              {showHint && <Hint>Hint</Hint>}
-              {showMessage && <Message validation={validation}>Message</Message>}
-            </Radio>
-          </Field>
-          <Field className="u-mt-xxs">
-            <Radio disabled={disabled} name="example">
-              <Label hidden={isHidden} isRegular={isRegular}>
-                Radio
-              </Label>
-              {showHint && <Hint>Hint</Hint>}
-              {showMessage && <Message validation={validation}>Message</Message>}
-            </Radio>
-          </Field>
+          <MD isBold>Radio group label</MD>
+          <RadioGroup aria-label="radio group label">
+            <Field>
+              <Radio disabled={disabled} name="example">
+                <Label hidden={isHidden} isRegular={isRegular}>
```

Comment:
`Label`s that participate in a `RadioGroup` should use context to reverse the font-weight and use regular weight by default. In other words, the only way to get them back to semibold would be to set `isRegular={false}`.

Reply:
Updated and added some tests for this.